README.md

```diff
@@ -113,6 +113,21 @@ script:
 ./tests/tests.sh
 ```
 
+# Lint
+
+To lint the shell scripts, you need to install `shellcheck` and `checkbashisms`. Those tools will check the scripts for issues and ensure they follow best practices.
+
+- On Debian-like systems: `apt-get install shellcheck devscripts`
```

Comment:
For Debian-like, the new method is `apt install shellcheck devscripts`, `apt-get` is not needed for installation anymore.

This is the only comment I have, everything else looks good!

```suggestion
- On Debian-like systems: `apt install shellcheck devscripts`
```

Reply:
Fixed

https://github.com/curl/wcurl/pull/32/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R120